app/assets/stylesheets/overrides.scss

```diff
@@ -212,3 +213,6 @@ table {
 .orange {
   color: $accent;
 }
+ul.off-canvas-list li a:hover {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3